pandas/core/indexes/base.py

```diff
@@ -2859,13 +2879,16 @@ def groupby(self, values):
 
         return result
 
-    def map(self, mapper):
-        """Apply mapper function to an index.
+    def map(self, mapper, na_action=None):
+        """Map values of Series using input correspondence (which can be a
```

Comment:
The first sentence should be a single line so that it looks OK in the autosummary table.

Reply:
done